src/univariate/continuous/normal.jl

```diff
@@ -182,18 +182,18 @@ end
 
 function suffstats(g::NormalKnownMu, x::AbstractArray{T}) where T<:Real
     μ = g.μ
-    s2 = abs2(x[1] - μ)
-    for i = 2:length(x)
+    s2 = 0
```

Comment:
Needs also something like
```suggestion
    s2 = zero(T) + zero(μ)
```
or
```suggestion
    s2 = abs2(zero(T) - zero(μ))
```